Kunena/Kunena-Forum
# Attachment manager limited to 30 (fix from rtv3)

Pull Request for Issue # .
#### Summary of Changes
#### Testing Instructions

This change add pagination into tab attachments in your profile, so if you have  more than 30 attachments you will see pagination
